src/ui/components/Card/index.js

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+
+import './Card.scss';
+
+
+export default class Card extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    footer: PropTypes.node,
+    header: PropTypes.node,
+  }
+
+  render() {
+    const { children, footer, header } = this.props;
+
+    return (
+      <div className="Card" ref={(ref) => { this.cardContainer = ref; }}>
+        {header ? (
+          <h2 className="Card-header" ref={(ref) => { this.header = ref; }}>
+            {header}
+          </h2>
+        ) : null}
+
+        {children ? (
+          <section className={classNames(
+            'Card-contents',
+            !footer ? 'no-footer' : '',
```

Comment:
These classes should be prefixed. Would it make sense to put them on the `Card` element instead? Either way, this can be written as:

```js
classNames('Card-contents', { 'Card--no-footer': !footer, 'Card--no-header': !header })